src/sagas/giphyRequests.js

```diff
@@ -0,0 +1,28 @@
+import { call, takeEvery, put } from 'redux-saga/effects';
+import { loadTrending, loadSearch } from '../api/giphy';
+import requests from '../actions/requests';
+
+export function* loadTrendingSaga() {
+	//console.log('[SAGA] loadTrendingSaga...');
+	try {
+		const giphyResponse = yield call(loadTrending);
+		yield put(requests.creators.gifsLoaded(giphyResponse));
+	} catch (error) {
+		yield put(requests.creators.gifsLoadError(error));
+	}
+}
+
+export function* loadSearchSaga(action) {
+	//console.log('[SAGA] loadSearchSaga...');
+	try {
+		const giphyResponse = yield call(() => loadSearch(action.text));
```

Comment:
You can pass arguments to the `call` effect. `call(loadSearch, [action.text])`, no need to create a function.